lib/insights.ts

```diff
@@ -37,6 +37,7 @@ import {
   setUserToken
 } from "./_cookieUtils";
 import { version } from "../package.json";
+import { supportsCookies } from "./utils";
```

Comment:
Why do we import it this file since we don't use it?

Reply:
great catch thanks! fixed here: 77eb366
I found out in the mean time that the rollup typescript plugin we use intentionally doesn't trigger compilation issues. 🙀 

I'll look into moving to https://github.com/ezolenko/rollup-plugin-typescript2.
Gave it a quick try. Quite a few things we've missed 🤦‍♂️